.github/workflows/build-vcpkg.yml

```diff
@@ -62,7 +62,7 @@ jobs:
             luajit: off
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         with:
           submodules: true
```

Comment:
Add `fetch-depth: 0` here and remove the unshallow step below (in several files)

Reply:
I accidentally approved 😭